usersimcrs/simulator/llm/interfaces/llm_interface.py

```diff
@@ -31,3 +32,18 @@ def generate_response(self, prompt: Prompt) -> Utterance:
             Utterance.
         """
         raise NotImplementedError()
+
+    @abstractmethod
+    def get_llm_response(self, prompt: str, **kwargs) -> str:
```

Comment:
I find this a bit confusing to have `generate_response` and `get_llm_response` -- based on the names the difference is not quite clear. Also, since it's supposed to be a high-level LLM interface, wouldn't it suffice to have the latter method that takes a prompt (plus optional arguments) and returns a string response and also use this for utterance generation directly?

Nit: the kwargs param is missing from the docstring

Reply:
As per discussion, I updated the name of the methods and the docstrings to make onjectives more explicit.